docs/jobs/openings/web3.md

```diff
@@ -0,0 +1,34 @@
+## Web3 Developer Opening at Aragon
+
+Aragon is an open source, decentralized project fighting for the freedom to organize. We believe that decentralized organizations can solve the world’s worst problems.
+
+We’re looking for a Web3 Developer to help us consolidate smart contracts with a frontend, we call this position The Glue. Our Web 3 Developer must not only be a Javascript rockstar but also a decentralization evangelist, someone who enjoys researching and testing new Web3 technologies.
+
+### Responsibilities
+
+- Take ownership of the development and maintenance of [aragon.js](https://github.com/aragon/aragon.js), the JavaScript library between smart contracts and the native web experience.
+- Research and experiment with new Web3 technologies, including but not limited to [Web3.js](https://github.com/ethereum/web3.js/) and [Parity systems](https://github.com/paritytech).
+- Assist on smart contract and frontend development when needed.
+- Help create developer tools and technical documentation.
+
+### Requirements
+
+- Track record working with Javascript environments.
+- Expert understanding on ES6/ES7, React, client-side storage, resource optimization, CSS, using processors such as LESS or SASS, RxJS, service workers, and device APIs.
+- Proven ability to ship quality, well-crafted code within tight deadlines.
+- Being passionate about open source and decentralization.
+- Being an autonomous individual with a proactive attitude and a love for time self-management. We work focus-driven, not hours-driven.
+
+### Good to have
+
+- Previous experience contributing or managing open source projects.
+- Previous experience working remotely.
+- Very good written communication skills.
+
+### How to apply
+
+- Write an email with the following attachments plus any information you consider important to *hiring@aragon.one*, with the subject `Web3 Developer Dev opening application`.
+- Please send along your GitHub profile. Since we are open source only, we require applicants for developer positions to have a very active and healthy GitHub profile.
```

Comment:
> we require applicants for developer positions to have a very active and healthy GitHub profile.

This is probably on all our positions, but I feel like we could relax this a little bit. Being excited by open source is not the same as having the opportunities / luxuries to work on a "healthy" Github profile. It's already a "good to have", which is enough, I think.

It'll be very obvious interacting with the person if they've never heard of Github or used anything open source.

Reply:
Yeah, as long as they have something to show, too, it won't matter too much. We have to make sure we're not excluding people based on how much time their previous job allowed them to work on open source :)

Reply:
Totally agree. Furthermore, I think we should update this line on all other job postings. It was inherited from the old job description and the truth is that most people don't really have healthy Github profiles.